src/main/java/com/puppycrawl/tools/checkstyle/checks/coding/AbstractSuperCheck.java

```diff
@@ -136,7 +136,7 @@ private boolean isSuperCallInOverridingMethod(DetailAST ast) {
      */
     private static boolean hasArguments(DetailAST methodCallDotAst) {
         final DetailAST argumentsList = methodCallDotAst.getNextSibling();
-        return argumentsList.getChildCount() > 0;
```

Comment:
Pitest [issue](https://circleci.com/gh/pbludov/checkstyle/3868#tests/containers/1) should be fixed.